hassio/src/addon-view/hassio-addon-audio.ts

```diff
@@ -0,0 +1,228 @@
+import "web-animations-js/web-animations-next-lite.min";
+
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import { HomeAssistant } from "../../../src/types";
+import { HassioAddonDetails } from "../../../src/data/hassio";
+import { PolymerChangedEvent } from "../../../src/polymer-types";
+import { hassioStyle } from "../resources/hassio-style";
+import { haStyle } from "../../../src/resources/styles";
+import { fireEvent } from "../../../src/common/dom/fire_event";
+
+interface Device {
+  device: string;
+  name: string;
+}
+
+@customElement("hassio-addon-audio")
+class HassioAddonAudio extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public addon!: HassioAddonDetails;
+
+  @property() public inputDevices!: Device[];
+  @property() public outputDevices!: Device[];
+  @property() public selectedInput!: string;
+  @property() public selectedOutput!: string;
```

Comment:
Shoudn't these be `private`?

Reply:
Yes, they should, have corrected that now.